zipkin4net/Criteo.Profiling.Tracing.UTest/Transport/T_ZipkinHttpTraceInjectorExtractor.cs

```diff
@@ -29,6 +31,18 @@ public void GetTraceThenSetHeadersEqualsOriginal(string encodedTraceId, string e
             Assert.AreEqual(flagsStr, recreatedHeaders[ZipkinHttpHeaders.Flags]);
         }
 
+        [Test]
+        public void Supports128BitsTraceId()
+        {
+            var traceId = "00FF0000000000001000000000002111";
```

Comment:
make a test that makes it obvious that the 00FF.. part is traceIdHigh? Ex by using shorter numbers like traceIdHigh=1, traceId=2 (as the hex is equiv to decimal and easy to see)